source/whats-new.txt

```diff
@@ -0,0 +1,168 @@
+.. _cpp-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade, driver
+
+Overview
+--------
+
+Learn what's new in:
+
+* :ref:`Version 3.10 <version-3.10>`
+* :ref:`Version 3.9 <version-3.9>`
+
+Breaking Change Types
+~~~~~~~~~~~~~~~~~~~~~
+
+New versions of the {+driver-short+} can introduce the following types of breaking changes:
+following groups:
+
+- **API** changes concern the API of the {+driver-short+} libraries. These changes are
+  accompanied by an API major version bump. You might need to edit C++ source code
+  in your project to address these changes.
+
+  .. example::
+
+     - Removed function ``func(A)``. Use ``func(A, B)`` instead.
+     - Changed redeclaration of ``bsoncxx::foo`` from ``v1::foo`` to ``v2::foo``. Qualify
+       ``bsoncxx::foo`` as ``bsoncxx::v1::foo`` to continue using the old interface.
+
+- **ABI** changes concern the ABI of the {+driver-short+} libraries. These changes are
+  accompanied by an ABI version bump, and may also be accompanied by an API major version
+  bump. You might need to rebuild your project or edit your C++ source code to address
+  these changes.
```

Comment:
Recommend including an important note here explicitly excluding ABI breaking changes for symbols declared in the `v_noabi` namespace from the ABI version bump requirement:

> [!IMPORTANT]
> ABI stability is not supported for symbols corresponding to entities declared in the `v_noabi` namespace. Breaking changes for such symbols are not necessarily accompanied by an ABI version bump.

I am undecided on whether breaking changes to the unstable ABI should even be documented on this page (since we are not committed to their stability in the first place), but if we do want to list them, then including this note will be important.

Reply:
My intuition is that it would be in our users' best interest to provide them with this information. We can always remove this note later if we decide to not include breaking changes to the unstable ABI on this page.